appbanners.info.yml

```
@@ -0,0 +1,6 @@
+name: App Banners
+description: Provides support for Apple Smart App Banners and Android native app install banners.
+type: module
+core: 8.x
+
+configure: appbanners.admin_settings
```

> All text files should end in a single newline (\n).
https://www.drupal.org/docs/develop/standards/coding-standards

```suggestion
configure: appbanners.admin_settings

```